src/dmd/cparse.d

```diff
@@ -1675,7 +1675,7 @@ final class CParser(AST) : Parser!AST
             auto stags = applySpecifier(stag, specifier);
             symbols.push(stags);
 
-            if (tt.tok == TOK.enum_)
+            if (0 && tt.tok == TOK.enum_)    // C11 proscribes enums with no members, but we allow it
```

Comment:
?

Reply:
ISO C forbids forward references to enum types, I guess we're picking and choosing what parts of the spec we're choosing to enforce?

Reply:
I wanted to leave it in in case we want to be strict C11.